### ISSUE TITLE
Implement latency measurements

### ISSUE THREAD

**Issue by rjhowell44** [canammex-tech/deepstream-services-library#293]
Need to measure Pipeline latency ... source-to-sink ... and make available as a new Source Info Display Type

**Comment by jlerasmus** [comment]
Agree, would be great to have latency measurements as with the sample deepstream-app.  Refer to: https://docs.nvidia.com/metropolis/deepstream/dev-guide/DeepStream_Development_Guide/baggage/group__ee__nvlatency__group.html

**Comment by rjhowell44** [comment]
Will target v0.08.alpha release for this
